BootloaderCorePkg/Library/PciEnumerationLib/PciEnumerationLib.c

```diff
@@ -1182,6 +1198,178 @@ EnablePciDevice (
   }
 }
 
+/**
+  Get a pointer of PCI resource allocation table
+
+  @param [out]  ResAllocTable   A table pointer for PCI resource allocation ranges
+
+ **/
+VOID
+GetPciResourceAllocTable (
+  OUT PCI_RES_ALLOC_TABLE       **ResAllocTable
+  )
+{
+  STATIC PCI_RES_ALLOC_TABLE    *TablePtr;

```

Comment:
EDK2 coding style does not allow static variable.  So consider moving it to global.

Reply:
Sure, I will move it to global.